src/Helpers/RelationHelper.php

```diff
@@ -176,6 +177,10 @@ public static function getModelRelations($model, bool $refresh = false): array
             return static::$relations[$class];
         }
 
+        if (empty($except) && method_exists($model, 'getExcludedRelations')) {
```

Comment:
Should this be a merge of the excepts, rather than a default fallback?

Reply:
yes, that's a good idea